tests/mock_data/tasking_data/get_quotations_decision_ACCEPTED.json

```diff
@@ -0,0 +1,147 @@
+{
+    "content": [
```

Comment:
suggestion but not necessary, maybe not in the scope of this PR (idk what the accepted pattern is): since there are harmless repeated values like created, updated, workspaceid, etc, perhaps consider creating a util function to generate these datas?

Reply:
Good point, I will create a ticket for this.

Reply:
DONE!